operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -216,16 +230,22 @@ public String submitExport(Instant since, List<String> types, Map<String, String
         if (since != null) {
             builder.fhirSearchFromDate(since.getValue().format(Instant.PARSER_FORMATTER));
         } else {
-            builder.fhirSearchFromDate("1970-01-01");
+            builder.fhirSearchFromDate("1970-01-01T00:00:00Z");
         }
 
         if (properties.get(BulkDataConstants.PARAM_TYPE_FILTER) != null) {
             builder.fhirTypeFilters(properties.get(BulkDataConstants.PARAM_TYPE_FILTER));
         }
 
-        builder.fhirExportFormat(properties.getOrDefault(BulkDataConstants.PARAM_OUTPUT_FORMAT, FHIRMediaType.APPLICATION_NDJSON));
 
         String entityStr = JobInstanceRequest.Writer.generate(builder.build(), true);
+        if (log.isLoggable(Level.FINE)) {
+            // TODO: need to remove secrets before we can log it. Clone the builder?
```

Comment:
I think I'm ok with uncommenting this code and removing the TODO

Reply:
I don't want to write any secrets like COS credentials to the log file.